lib/helpers/deleteObjectHelpers.js

```diff
@@ -0,0 +1,80 @@
+const { messages } = require('elasticio-node');
+const createObj = require('../../lib/actions/createObject.js');
+const deleteObjectData = require('../../spec-integration/actions/deleteObject.json');
+
+/**
+ * Des: 
+ * - This function will use the active sf connection to delete an obj by Id
+ * - Implemented in a try catch block and will emit the salesforce error if
+ * it exists 
+ * @implementedBy C.J.V
+ * @param sfConn an open salesforce connection
+ * @param id the id of the obj to delete
+ * @param objType the obj type to delete
+*/
+module.exports.deleteObjById = async function deleteObjById(sfConn, id, objType) {
+  try {
+    this.logger.trace(`Here's the ID of what we're deleting ${id}`);
```

Comment:
Need to change log level to `info` here. Also, please, remove useless spaces in this script